### ISSUE TITLE
Cleanup - remove all the join and conditional classes and use the new utils

### ISSUE THREAD

**Issue by daniele-zurico** [Capgemini/dcx-react-library#407]
Two new utilities has been created for multiple classes and contional classes in `/src/common/utils/classNames.ts`
- classNames
- conditionalClassNames
Where the first one is for simple usages with scenarios like:
```jsx
<div className={['a','b'].join(' ')}/>
```
and the second one instead:
```jsx
<div className={['a', condition ? 'b' : 'c'].join(' ')]}
```
the usage will become:
```jsx
<div className={classNames(['a','b'])}/>
```
and
```jsx
<div className={['a', { 'a' : condition, 'b': !condition}]]}
```
To see one example already implemented goes to `/src/formSelect/FormSelect.tsx` (line 196)

Scan all the components and when you find conditional classnames or concatenated classNames use the 2 explained functions

**Comment by daniele-zurico** [comment]
Closed by #411

**Comment by daniele-zurico** [comment]
Closed by #411
